src/createEnvfile.ts

```diff
@@ -17,6 +17,7 @@ export interface ICreateEnvFileParams
   type: EnvFileType | string;
   outputDir: string;
   secretIds: string[];
+  outputWhitelist?: string[];
```

Comment:
I would like you to add outputWhitelist to the README parameter description🐱

Can you do it?

Reply:
@keitakn 
I add description of outputWhitelist to README🐱